app/scripts/services/brackets.coffee

```diff
@@ -9,6 +9,13 @@
 ###
 angular.module('edudashAppSrv').service 'bracketsSrv', ->
 
+  getBrackets: (metric) ->
```

Comment:
this is just to color the three parts of the charts, we won't use it for comparing